src/wallet/spend.cpp

```diff
@@ -444,8 +444,14 @@ std::vector<OutputGroup> GroupOutputs(const CWallet& wallet, const std::vector<C
     return groups_out;
 }
 
-std::optional<SelectionResult> AttemptSelection(const CWallet& wallet, const CAmount& nTargetValue, const CoinEligibilityFilter& eligibility_filter, std::vector<COutput> coins,
+std::optional<SelectionResult> AttemptSelection(const CWallet& wallet, const CAmount& nTargetValue, const CoinEligibilityFilter& eligibility_filter, const AvailableCoins& availableCoins,
                                const CoinSelectionParams& coin_selection_params)
+{
+    std::optional<SelectionResult> result = ChooseSelectionResult(wallet, nTargetValue, eligibility_filter, availableCoins.all(), coin_selection_params);
```

Comment:
In f42db05a0ea8a4acfaf8e0117824cfd0f1d0547b "refactor: use AvailableCoins struct in SelectCoins"

The commit message mentions a newly introuduced method `ChooseSelectionAlgorithm`, but the code introduces `ChooseSelectionResult`.